x-pack/plugins/enterprise_search/public/applications/app_search/app_logic.ts

```diff
@@ -26,5 +33,93 @@ export const AppLogic = kea<MakeLogicType<IAppValues, IAppActions>>({
         initializeAppData: () => true,
       },
     ],
+    account: [
+      {} as IAccount,
+      {
+        initializeAppData: (_, { appSearch: account }) => account,
+        setOnboardingComplete: (state) => ({
+          ...state,
+          onboardingComplete: true,
+        }),
+      },
+    ],
+    configuredLimits: [
+      {} as IConfiguredLimits,
+      {
+        initializeAppData: (_, { configuredLimits }) => configuredLimits.appSearch,
+      },
+    ],
+    ilmEnabled: [
+      false,
+      {
+        initializeAppData: (_, { ilmEnabled }) => !!ilmEnabled,
+      },
+    ],
+  },
+  selectors: {
+    /**
+     * Massages the `role` data we receive from the Enterprise Search
```

Comment:
Strong opinion I have held for 5 seconds: we should find a better word than "massages".  We're rubbing the data's back? Lighting some candles and putting on some music? Maybe just "converts" or "transforms"

Reply:
Hahaha I had that exact same moment of indecision but then YOLO'd into it. I tend like "transmogrifies data" as well but i think people find that one too bombastic :trollface:  I'll switch to "transforms"

Reply:
Addressed in de93ff9